src/index.js

```diff
@@ -31,12 +31,13 @@ const createCharacters = () => {
 const App = () => {
   return (
     <div>
+      <RateButtons>
         <CharacterCard
           name={createCharacters().name}
           title={createCharacters().title}
           image={createCharacters().image}
         />
-        <RateButtons />
+        </RateButtons>
```

Comment:
I'm a little confused about the `<RateButtons />` component. Would you mind explaining what the rate button is composed of? It might be worth renaming, to make it more clear